components/status-indicator/status-indicator.js

```diff
@@ -0,0 +1,99 @@
+import '../colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+
+class StatusIndicator extends LitElement {
+
+	static get properties() {
+		return {
+			/**
+			 * State of status indicator to display. Valid values are: 'default', 'success', 'alert', and 'none'.
+			 */
+			state: {
+				type: String,
+				reflect: true
+			},
+			/**
+			 * Text for status indicator.
+			 */
+			text: {
+				type: String,
+				value: null
+			},
+			/**
+			 * Use when the status is very important and needs to have a lot of prominence.
+			 */
+			bold: {
+				type: Boolean,
+				value: false

```

Comment:
Similarly, the `value` should be set in the constructor. But this should be reflected since you're using CSS attribute selectors that require it.